schemas/questionnaire_flow/linear.json

```diff
@@ -4,26 +4,19 @@
   "options": {
     "type": "object",
     "properties": {
-      "include_summary": {
-        "description": "Whether to include the summary of answers",
-        "type": "boolean"
-      },
-      "collapsible": {
-        "description": "When True, the summary page will contain a list of collapsible section summaries",
-        "type": "boolean"
+      "summary": {
+        "type": "object",
+        "description": "Options for the summary",
```

Comment:
I think the description should make it clear what the `summary` property is, something like:
```
Using this property will include a summary on the submit page.
```